content/global/inclusive.mdx

```diff
@@ -0,0 +1,111 @@
+---
+title: Inclusive guidelines
+---
+
+Inclusive design is a methodology that allows people of different backgrounds and experiences to be productive and express their identities in a digital space without fear of compromising their personal safety.
+
+## Starting a new project
+
+As you begin tackling a new design problem with your team, here are a few core considerations to help guide you in your preliminary research.
+
+### Think through key questions
+
+It's important to consider different perspectives from the very beginning, before jumping into any solutions. These initial questions will help you be **intentional and proactive**, rather than reactive later on.
+
+Going through these questions is most effective within a large brainstorming session with your team, where everyone can bring their backgrounds and experiences to the table. You can also continue to ask these questions throughout the design process, especially as scope changes.
+
+#### How can this feature be used to harm someone?
```

Comment:
I know the heading before is "Think through key questions", but I wonder if we need to be more explicit that this is where the questions are starting. Would adding: "Question 1" before this heading be too much? Or something similar that makes this clear?